presto-product-tests/src/main/java/io/prestosql/tests/utils/QueryAssertions.java

```diff
@@ -20,15 +20,32 @@
 
 import java.util.function.Supplier;
 
-import static io.prestosql.testing.assertions.Assert.assertEventually;
 import static java.lang.String.format;
 import static org.testng.Assert.fail;
 
 public final class QueryAssertions
 {
     public static void assertContainsEventually(Supplier<QueryResult> all, QueryResult expectedSubset, Duration timeout)
     {
-        assertEventually(timeout, () -> assertContains(all.get(), expectedSubset));
```

Comment:
why so?

Reply:
It is copied from Presto main

Reply:
Oh... It is copied below. So no need to inline it.